1-js/13-modules/02-import-export/article.md

```diff
@@ -97,7 +97,7 @@ Well, there are few reasons.
 2. Explicit list of imports gives better overview of the code structure: what is used and where. It makes code support and refactoring easier.
 
 ```smart header="Don't be afraid to import too much"
-Modern build tools, such as [webpack](https://webpack.js.org/) and others, bundle modules together and optimize them to speedup loading. They also removed unused imports.
+Modern build tools, such as [webpack](https://webpack.js.org/) and others, bundle modules together and optimize them to speedup loading. They also remove unused imports.
```

Comment:
We already have a PR for this https://github.com/javascript-tutorial/en.javascript.info/pull/3326